suites/quincy/rbd/rbd-upstream-cli-operations.yaml

```diff
@@ -0,0 +1,150 @@
+# Tier0: RBD build evaluation
```

Comment:
Likewise, IMO the filename could be `basic-operations_unit_testing.yaml`

Reply:
Updated as per comments